elements/pfe-accordion/src/_shared-assets.scss

```diff
@@ -1,4 +1,3 @@
-// NOTE: pfe-accordion uses shared styles that can be found in `pfe-sass/mixins/_components.scss`
```

Comment:
did this accidentally get removed?

Reply:
Dang yes, I was testing adding something into this file and must have Ctrl+Z'ed wrong...  I'll open a smolboi to bring it back.